client/packages/common/src/ui/components/navigation/ListOptions/ListOptions.tsx

```diff
@@ -55,22 +55,28 @@ export const ListOptions = ({
             onClick={() => onClick(option.id)}
           >
             <ListItemIcon sx={{ padding: 0, minWidth: 25 }}>
-              {enteredLineIds?.includes(option.id) ? (
-                startIcon
-              ) : (
-                <Box style={{ visibility: 'hidden' }}>{startIcon}</Box>
-              )}
+              <Box
+                style={{
+                  visibility: enteredLineIds?.includes(option.id)
```

Comment:
If this is in response to my comment I was more meaning that for this requisition of 21 items

![image](https://github.com/user-attachments/assets/b219f49b-0e23-4fe1-90ae-49397e2a468b)

It renders the checkbox component 63 times (3 times per row for whatever reason 😁) when only one row is actually showing the checkbox. Where as if you had 

```jsx
              <Box>{enteredLineIds?.includes(option.id) && startIcon}</Box>
```

Instead it only renders a checkbox 3 times, for the one row it's actually displayed on

Reply:
Oh no it was just a refactor.

I have both icons hidden because I didn't want it to move the text when it appears... which happens...

Reply:
Right I had to fiddle around a bit to see only seems to happen on a small window, so probably would happen on tablet!